Milestone3/backend/models/recipeSchema.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+
+const recipeSchema = new mongoose.Schema({
+    name: [String],
```

Comment:
So here you would probably just want type String instead of [String], I believe if you use the brackets it will be interpreted as an array of strings which is probably not what you want